firedrake/randomfunctiongen.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
+import inspect
+import randomgen
+
+from firedrake.function import Function
+from ufl import FunctionSpace
+
+
+"""
+
+This module wraps 'randomgen' and enables
```

Comment:
This documentation doesn't show up in Sphinx. I think it needs to be the first thing in the file.